fontbe/src/name.rs

```diff
@@ -25,14 +27,18 @@ impl Work<Context, AnyWorkId, Error> for NameWork {
     }
 
     fn read_access(&self) -> Access<AnyWorkId> {
-        Access::Variant(AnyWorkId::Fe(FeWorkId::StaticMetadata))
+        AccessBuilder::new()
+            .variant(AnyWorkId::Fe(FeWorkId::StaticMetadata))
+            .variant(AnyWorkId::Be(WorkId::ExtraFeaTables))
```

Comment:
This is an unfortunate dependency; fea is often long pole so blocking things on fea is undesirable. I suppose to avoid you'd have to fix name in final font assembly which seems even worse.